plugins/hub/osbuild.py

```diff
@@ -28,8 +28,21 @@
             "description": "Distribution"
         },
         {
-            "type": "string",
-            "description": "Image Type",
+            "oneOf": [
+                {
+                    "type": "string",
+                    "description": "Image Type"
+                },
+                {
+                    "type": "array",
+                    "description": "Image Types",
```

Comment:
Can you add a note that this option is deprecated? JSON Schema also supports `deprecated` annotation but it was added fairly recently so I'm worried it may blow up, see https://json-schema.org/draft/2019-09/json-schema-validation.html#rfc.section.9.3

Reply:
Oh yeah, thanks for pointing this out 👍

Reply:
Done